my-app/src/components/memberCard/MemberCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faUser,
+  faCheck,
+} from "@fortawesome/free-solid-svg-icons";
+import "./MemberCard.css";
+
+export const MemberCard = ({ name, selected }) => {
+  return (
+    <div className={`member-card ${selected ? "selected" : "not-selected"}`}>
```

Comment:
Instead of toggling between selected and not-selected classes, you can have a single member-card class and conditionally apply on class maybe call `is-selected.` This will keep the styles more focused on specific states without requiring both classes.

`<div className={`member-card ${selected ? "is-selected" : ""}`}>
`

```
.member-card {
    /* Common styles */
}

.member-card.is-selected {
    /* is-selected styles */
}
```